addons/website_livechat/static/tests/tours/website_livechat_chatbot_flow.js

```diff
@@ -248,8 +248,7 @@ registry.category("web_tour.tours").add("website_livechat_chatbot_flow_tour", {
                 trigger: messagesContain("I will transfer you to a human."),
             },
             {
-                trigger:
-                    ".o-livechat-root:shadow .o-mail-NotificationMessage:contains(joined the channel)",
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input:not(:disabled)",
```

Comment:
Nitpick: `:not(:disabled)` is equivalent to `:enabled`, which is more readable.

Reply:
It's a rare case I would put a comment / `content` here because the assertion is not that clear in the intention. Let's say livechat is rewritten from scratch and we keep the assertion, why checking composer enabled here is relevant? Could give a hint this wait to operator being added to conversation.

_(I know there's the assertion in python, but this is not good to understand this step by looking in a later steps in LOCs in a different file, the comment/`content` remedy to this problem)_